pkg/envoy/api/cluster.go

```diff
@@ -20,6 +20,8 @@ import (
 	"time"
 
 	envoyCluster "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
+	envoy_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
```

Comment:
[Package names with snake or camel case syntax dont look good in Go](https://go.dev/blog/package-names) so I suggest we avoid it for alias names too.
One suggestion is to use lower case letters (eg. used in Serving).
In any case we should have some consistency and dont mix styles.